### ISSUE TITLE
Setting up the transfer learning

### ISSUE THREAD

**Issue by PatrickAvis** [SeanPLeary/dc_tts-transfer-learning#2]
Hi,

I'm looking at performing transfer learning but am rather confused as to where to start. I assume I train my own model (or use a pre-trained one) and then replace the audio samples with my own...but then what?

I would imagine I need to bring in the weights and the network but unsure what I have to do to get it to work, Are you able to run through in a few steps?

Many thanks
Patrick

**Comment by yupitsaditya** [comment]
Yeah, there seems to be a folder named "private" which wasn't uploaded to Github, which makes perfect sense. But if someone can provide the basic structure of those folders, that would be of great help.
